docs/DatasourcesContents.md

```diff
@@ -0,0 +1,69 @@
+- [shape](https://github.com/oracle/terraform-provider-oci/tree/master/docs/datasources/core/shape.md)
+- [subnet](https://github.com/oracle/terraform-provider-oci/tree/master/docs/datasources/core/subnet.md)
+- [virtual_networks](https://github.com/oracle/terraform-provider-oci/tree/master/docs/datasources/core/virtual_networks.md)
+- [vnic_attachments](https://github.com/oracle/terraform-provider-oci/tree/master/docs/datasources/core/vnic_attachments.md)
+- [vnic](https://github.com/oracle/terraform-provider-oci/tree/master/docs/datasources/core/vnic.md)
+- [volume_attachments](https://github.com/oracle/terraform-provider-oci/tree/master/docs/datasources/core/volume_attachments.md)
+- [volume_backups](https://github.com/oracle/terraform-provider-oci/tree/master/docs/datasources/core/volume_backups.md)
+- [volumes](https://github.com/oracle/terraform-provider-oci/tree/master/docs/datasources/core/volumes.md)
+
+## database
+- [database](https://github.com/oracle/terraform-provider-oci/tree/master/docs/datasources/database/database.md)
+- [databases](https://github.com/oracle/terraform-provider-oci/tree/master/docs/datasources/database/databases.md)
+- [db_home](https://github.com/oracle/terraform-provider-oci/tree/master/docs/datasources/database/db_home.md)
+- [db_homes](https://github.com/oracle/terraform-provider-oci/tree/master/docs/datasources/database/db_homes.md)
+- [db_node](https://github.com/oracle/terraform-provider-oci/tree/master/docs/datasources/database/db_node.md)
+- [db_nodes](https://github.com/oracle/terraform-provider-oci/tree/master/docs/datasources/database/db_nodes.md)
+- [db_system_shapes](https://github.com/oracle/terraform-provider-oci/tree/master/docs/datasources/database/db_system_shapes.md)
+- [db_systems](https://github.com/oracle/terraform-provider-oci/tree/master/docs/datasources/database/db_systems.md)
+- [db_versions](https://github.com/oracle/terraform-provider-oci/tree/master/docs/datasources/database/db_versions.md)
+
+## identity
+- [api_key](https://github.com/oracle/terraform-provider-oci/tree/master/docs/datasources/identity/api_key.md)
+- [availability_domain](https://github.com/oracle/terraform-provider-oci/tree/master/docs/datasources/identity/availability_domain.md)
+- [compartment](https://github.com/oracle/terraform-provider-oci/tree/master/docs/datasources/identity/compartment.md)
+- [group](https://github.com/oracle/terraform-provider-oci/tree/master/docs/datasources/identity/group.md)
+- [policy](https://github.com/oracle/terraform-provider-oci/tree/master/docs/datasources/identity/policy.md)
+- [swift_password](https://github.com/oracle/terraform-provider-oci/tree/master/docs/datasources/identity/swift_password.md)
+- [user](https://github.com/oracle/terraform-provider-oci/tree/master/docs/datasources/identity/user.md)
+-
+[user_group_membership](https://github.com/oracle/terraform-provider-oci/tree/master/docs/datasources/identity/user_group_membership.md)
+
+
+## loadbalancer
```

Comment:
Should these titles be Load Balancer and Object Storage?

Reply:
will check it

Reply:
I took all the category names from the actual folder names in each branch - just for consistency. Personally, I'd capitalize each and separate combined words!